dotCMS/src/main/java/com/dotcms/jobs/business/util/JobUtil.java

```diff
@@ -160,4 +183,39 @@ public static <T extends JobEvent> void sendEvents(final T event) {
                 .onFailure(e -> Logger.error(JobUtil.class, e.getMessage()));
     }
 
+    /**
+     * Converts the given object into a Map representation.
+     *
+     * @param toTransform The object to be transformed into a Map. It should be serializable to a
+     *                    JSON string.
+     * @return A Map containing the data of the given object. If the object cannot be serialized or
+     * its representation is empty, an empty Map is returned.
+     * @throws JsonProcessingException If the object cannot be converted to a JSON string.
+     */
+    public static Map<String, Object> transformToMap(final Object toTransform)
+            throws JsonProcessingException {
+
+        final var asString = transformToString(toTransform);
+        if (!UtilMethods.isSet(asString)) {
+            return Map.of();
+        }
+
+        JSONObject resultJsonObject = new JSONObject(asString);
```

Comment:
this is a bad pattern you may be able to convert the object to map from jersey, this is spending so much cpu time and mem

Reply:
Fixed, now we are using `objectMapper.get().convertValue(toTransform, Map.class)`.